package-parser/package_parser/processing/migration/_differ.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABC, abstractmethod
+
+from package_parser.processing.api.model import Class, Function, Parameter, Result
+
+
+class AbstractDiffer(ABC):
+    @staticmethod
+    @abstractmethod
+    def diff_attributes(
+        attributes_a: list[str],
```

Comment:
To be in line with the other methods we should only expect two attributes to compare rather than two lists of attributes. Otherwise, we'd also need change something like diff_classes to take a list of classes since we do not know a priori which classes match - this is what this class is for after all.